docs/content/_index.md

```diff
@@ -0,0 +1,116 @@
+[cascade]
+    type = 'docs'
++++
+
+# Documentation for `hugo-python-distributions`
+
+This webpage is meant to serve the documentation for the `hugo-python-distributions` project, which aims to provide a distribution
+channel for the extended version of the Hugo static site generator through `pip`-installable binaries (wheels) hosted on the [Python Package Index (PyPI)](https://pypi.org/).
+
+## Table of contents
+
+- [Quickstart](#quickstart)
+- [Supported platforms](#supported-platforms)
+- [Building from sources](building-from-sources/)
+  - [Cross-compiling for different architectures](#cross-compiling-for-different-architectures)
+    - [macOS](#macos)
+    - [Linux](#linux)
+    - [Windows](#windows)
+
+
+### Quickstart
+
+Create a [virtual environment](https://realpython.com/python-virtual-environments-a-primer/) and install the package (or install it globally on your system):
+
+{{< tabs items="Linux/macOS,Windows" >}}
+
+  {{< tab >}}
+    ```bash
+    python -m virtualenv venv
+    source venv/bin/activate
+    python -m pip install hugo
+    ```
+  {{< /tab >}}
+
+  {{< tab >}}
+    ```powershell
+    py -m virtualenv venv
+    venv\Scripts\activate.bat
+    py -m pip install hugo
+    ```
+  {{< /tab >}}
+
+{{< /tabs >}}
+
+This example is using [`virtualenv`](https://virtualenv.pypa.io/en/latest/) to create a virtual environment, however, you can use any virtual environment manager of your choice. Some popular ones are the built-in [`venv`](https://docs.python.org/3/library/venv.html) module, [`virtualenvwrapper`](https://virtualenvwrapper.readthedocs.io/en/latest/), [`pipenv`](https://pipenv.pypa.io/en/latest/), [`conda`](https://docs.conda.io/en/latest/), [`poetry`](https://python-poetry.org/), [`pyenv`](https://github.com/pyenv/pyenv), [`uv`](https://github.com/astral-sh/uv), and so on.
+
+This places a `hugo` installation with an executable in your virtual environment and adds an [entry point](https://packaging.python.org/en/latest/specifications/entry-points/) to it in your virtual environment's `bin` directory.
+
+Then, you can use the `hugo` CLI commands as you would normally:
+
+```bash
+hugo version
+hugo env --logLevel info
+hugo new site my-new-site
+hugo mod <...>
+hugo --printI18nWarnings --buildDrafts server
+```
+
+and more!
+
+Alternatively, you can install the package globally on your system:
+
+{{< tabs items="Linux/macOS,Windows" >}}
+
+  {{< tab >}}
+    ```bash
+    python3.X -m pip install hugo
+    ```
+  {{< /tab >}}
+
+  {{< tab >}}
+    ```powershell
+    py -m pip install hugo
+    ```
+  {{< /tab >}}
+
+{{< /tabs >}}
+
+{{< callout emoji=✨ >}}
+
+It is a great idea to use [`pipx`](https://github.com/pypa/pipx) to install or use Hugo in an isolated location without having to create a virtual environment, which will allow you to run Hugo as a command-line tool without having to install it globally on your system.
+
+{{</ callout >}}
+
+
+{{< tabs items="Install it as an app globally, Run it" >}}
```

Comment:
Although it looks good but I personally think that tabs are good to divide platform dependent instructions but when it comes to this type of sections I prefer them to be straight & simple. 
Tho it's just a thought not a suggestion as it looks nice as well.

Reply:
Good idea, I think too many tabs are confusing for the user. I have a lot to restructure on this page but I don't wish to rewrite the documentation again (or if I do, I wish to keep it minimal). I'll keep this conversation unresolved for now, thanks for the initial review! I plan to return to this soon enough